pkg/handler/v1/node/node.go

```diff
@@ -557,13 +555,10 @@ func convertNodeMetrics(metrics []v1beta1.NodeMetrics, availableResources map[st
 		nodeMetric := apiv1.NodeMetric{
 			Name: m.Name,
 		}
-		err := scheme.Scheme.Convert(&m.Usage, &usage, nil)
-		if err != nil {
-			return nil, err
-		}
+
```

Comment:
Default conversions are not supported in Kubernetes 1.19+ anymore. In all cases in our codebase, the conversion was merely a convoluted DeepCopy() (I think, no offense).